pom.xml

```diff
@@ -153,8 +153,10 @@
         <hive.archive.name>apache-hive-${hive.engine.hive.version}-bin.tar.gz</hive.archive.name>
         <hive.archive.mirror>${apache.archive.dist}/hive/hive-${hive.engine.hive.version}</hive.archive.mirror>
         <hive.archive.download.skip>false</hive.archive.download.skip>
+        <httpcore.version>4.4.15</httpcore.version>
         <httpclient.version>4.5.14</httpclient.version>
```

Comment:
Since it is not used, we should delete `httpclient.version`.

Reply:
Search again. jdbc driver is still using HC4, as mentioned in PR description.